testhelpers/binaries.go

```diff
@@ -0,0 +1,76 @@
+package testhelpers
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+func ConditionallyBuildLifecycle(t *testing.T, dir string, goos string) {
```

Comment:
Do we need to add it to launcher_test.go?

Reply:
(see above)